ext/openssl/ossl_x509cert.c

```diff
@@ -440,7 +440,11 @@ ossl_x509_set_not_before(VALUE self, VALUE time)
 
     GetX509(self, x509);
     asn1time = ossl_x509_time_adjust(NULL, time);
+#if !defined(LIBRESSL_VERSION_NUMBER) && (OPENSSL_VERSION_NUMBER >= 0x10100000L)
+    if (!X509_set1_notBefore(x509, asn1time)) {
+#else
     if (!X509_set_notBefore(x509, asn1time)) {
+#endif
```

Comment:
I'd define macro in ext/openssl/openssl_missing.h. It's actually done for the getter functions, but apparently I forgot to do that for the setters.

    #define X509_set1_notBefore(x, t) X509_set_notBefore(x, t)